src/webgl.nim

```diff
@@ -436,7 +436,7 @@ proc getBoundingClientRect*(c:Canvas):tuple[top,bottom,left,right:float] =
   """}
   result = (t,b,lf,r)
 
-proc log*(str:varargs[untyped]) = {.emit: "console.log(`str`);".}
+proc log*(str: string) = echo str
```

Comment:
First of all, thank you for the PR! 
This would change the behaviour of `log` (it would only accept a single `string` input instead of a list), and I believe a more appropriate way to log is now using `console.log` from jsffi.  
Could you change the type to `varargs[string]` and also deprecate this proc?